server/athenian/api/application.py

```diff
@@ -906,26 +907,27 @@ def raise_graceful_exit():
         loop.add_signal_handler(signal.SIGTERM, raise_graceful_exit)
         os.kill(os.getpid(), signal.SIGTERM)
 
-    def add_api(self, specification: str, **kwargs):
+    def add_api(self, specification: str, ref_resolver_store: Optional[dict] = None, **kwargs):
         """Load the API spec and add the defined routes."""
-        api = super().add_api(specification, **kwargs)
+        api = super().add_api(specification, ref_resolver_store=ref_resolver_store, **kwargs)
         api.subapp["aiohttp_jinja2_environment"].autoescape = False
         api.jsonifier.json = FriendlyJson
         for k, v in api.subapp.items():
             self.app[k] = v
         api.subapp._state = self.app._state
-        components = api.specification.raw["components"]
-        components["schemas"] = dict(sorted(components["schemas"].items()))
```

Comment:
This is needed for the UI to display the components in the proper order. They are in the document order by default.

Reply:
ops, fixed